test/unit/specs/components/staking/__snapshots__/PageDelegateIndex.spec.js.snap

```diff
@@ -213,11 +227,139 @@ exports[`PageDelegate has the expected html structure 1`] = `
+              >
+                <!---->
+              </div>
+              <div
+                class="ni-li-dl"
+              >
+                <div
+                  class="ni-li-dt"
+                >
+                  Voting Power
+                </div>
+                <div
+                  class="ni-li-dd"
+                >
+                  10000 ATOM
+                </div>
+              </div>
+            </div>
+          </div>
+          <div
+            class="ni-li"
+          >
+            <div
+              class="ni-li-container"
+            >
+              <div
+                class="ni-li-thumb"
+              >
+                <!---->
+              </div>
+              <div
+                class="ni-li-dl"
+              >
+                <div
+                  class="ni-li-dt"
+                >
+                  Bonded Atoms
+                </div>
+                <div
+                  class="ni-li-dd"
+                >
+                  5000 ATOM
+                </div>
+              </div>
+            </div>
+          </div>
+          <div
+            class="ni-li"
+          >
+            <div
+              class="ni-li-container"
+            >
+              <div
+                class="ni-li-thumb"
+              >
+                <!---->
+              </div>
+              <div
+                class="ni-li-dl"
+              >
+                <div
+                  class="ni-li-dt"
+                >
+                  Commission
+                </div>
+                <div
+                  class="ni-li-dd"
+                >
+                  NaN%
+                </div>
+              </div>
+            </div>
+          </div>
+          <div
+            class="ni-li"
           >
-            <pre>
-              
-            </pre>
+            <div
+              class="ni-li-container"
+            >
+              <div
+                class="ni-li-thumb"
+              >
+                <!---->
+              </div>
+              <div
+                class="ni-li-dl"
+              >
+                <div
+                  class="ni-li-dt"
+                >
+                  Max Commission
+                </div>
+                <div
+                  class="ni-li-dd"
+                >
+                  NaN%
+                </div>
+              </div>
+            </div>
+          </div>
+          <div
+            class="ni-li"
+          >
+            <div
+              class="ni-li-container"
+            >
+              <div
+                class="ni-li-thumb"
+              >
+                <!---->
+              </div>
+              <div
+                class="ni-li-dl"
+              >
+                <div
+                  class="ni-li-dt"
+                >
+                  Max Commission Increase
+                </div>
+                <div
+                  class="ni-li-dd"
+                >
+                  NaN%
```

Comment:
Is this intended?

Reply:
It's intended, because we still don't have commission data.